pkg/controller/service/service_controller.go

```diff
@@ -339,6 +339,10 @@ func (s *ServiceController) syncLoadBalancerIfNeeded(service *v1.Service, key st
 		}
 		newStatus, err = s.ensureLoadBalancer(service)
 		if err != nil {
+			if err == cloudprovider.NotImplemented {
```

Comment:
The NotImplemented error is an already existing error which cloud providers outside of tree may be using. I do not think we should ignore it here. If we want an error to indicate the functionality is implemented  elsewhere lets generate a new error for that case.

Reply:
ack, will add a new error.